config/samples/csiaddons_v1alpha1_networkfenceclass.yaml

```diff
@@ -6,4 +6,7 @@ metadata:
     app.kubernetes.io/managed-by: kustomize
   name: networkfenceclass-sample
 spec:
-  # TODO(user): Add fields here
+  provisioner: driver.csi.com
+  parameters:
+    csiaddons.openshift.io/networkfence-secret-name: secret
+    csiaddons.storage.openshift.io/network-secret-namespace: default
```

Comment:
is that correct? One with `.storage.` in and one not?

Reply:
its a typo fixed it.